### DISCUSSION TITLE
Possible to create the same reaction twice on an issue comment

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#62385] [Mobile]
https://github.com/cli/cli/issues/7761#issuecomment-1660237478

Android Mobile beta

## steps

1. Open an issue from notifications
2. Tap the blank emoji below a comment
3. Tap 👀
4. The picker disappeared but the emoji didn't appear
5. Repeat 2-3
6. The emoji appeared twice

![Screenshot_20230801-085313.png](https://github.com/community/community/assets/2119212/4527ac4b-3ae4-4aec-8561-a78c80967704)

**Comment by mxie** [comment]
Thanks for letting us know @jsoref! Is this only happening when you try to access the issue from a notification? Also, is this from the notifications Inbox or is it from a push notification?

**Reply by jsoref** [reply]
I doubt it's related to how I accessed it. I think I initially accessed the issue from an android notification and later from the notifications inbox.

The key is that the update of the reactions thing after the first step 3 took too long. But obviously some code could be added to the frontend to make sure it doesn't do this since it'd never be right to have two sections for the same reaction.
